provider/resource_keycloak_group_memberships_test.go

```diff
@@ -411,6 +428,38 @@ resource "keycloak_group_memberships" "group_members" {
 	`, realm, group, username)
 }
 
+func testKeycloakGroupMemberships_moreThan100members(realm, group string) string {
+	count := 200
+	var usersInGroupInterpolated []string
+	for i := 0; i < count; i++ {
+		usersInGroupInterpolated = append(usersInGroupInterpolated, fmt.Sprintf("${keycloak_user.users.%d.username}", i))
+	}
+	return fmt.Sprintf(`
+resource "keycloak_realm" "realm" {
+        realm = "%s"
+}
+
+resource "keycloak_group" "group" {
+        name     = "%s"
+        realm_id = "${keycloak_realm.realm.id}"
+}
+
+resource "keycloak_user" "users" {
+	count = %d
+        realm_id = "${keycloak_realm.realm.id}"
+        username = "terraform-user-${count.index}"
+}
+
+resource "keycloak_group_memberships" "group_members" {
+        realm_id = "${keycloak_realm.realm.id}"
+        group_id = "${keycloak_group.group.id}"
+
+        members = %s
```

Comment:
I think you should be able to use a [splat expression](https://www.terraform.io/docs/configuration/expressions.html#splat-expressions) here instead of building a list of usernames manually.

```suggestion
        members = keycloak_user.users.*.username
```

Reply:
done